README.md

```diff
@@ -66,23 +66,20 @@ firectl \
 Getting Started on AWS
 ---
 
-- Create an `i3.metal` instance using Amazon Linux 2
-- Build latest version of firectl:
+- Create an `m5d.metal` instance using Amazon Linux 2
+- Get firectl binary:
 
   ```
-  sudo yum install -y git
-  git clone https://github.com/firecracker-microvm/firectl
-  sudo amazon-linux-extras install -y golang1.11
-  cd firectl
-  make
+  curl -oL firectl https://firectl-release.s3.amazonaws.com/firectl-v0.1.0
```

Comment:
Can you include a step to validate the binary was not corrupted before marking it executable?  We publish the expected sha256 hash at https://firectl-release.s3.amazonaws.com/firectl-v0.1.0.sha256.